cg/meta/orders/ticket_handler.py

```diff
@@ -151,3 +154,34 @@ def replace_empty_string_with_none(cls, obj: Any) -> Any:
                 else:
                     obj[key] = cls.replace_empty_string_with_none(item)
         return obj
+
+    def create_trimmed_new_ticket_message(
```

Comment:
Why do we add the prefix and then remove it??

Reply:
Restructured so that different headers are added depending on new or connecting order